go.mod

```diff
@@ -3,32 +3,31 @@ module github.com/ipfs/go-bitswap
 require (
 	github.com/cskr/pubsub v1.0.2
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.2 // indirect
-	github.com/google/uuid v1.1.1
+	github.com/google/uuid v1.1.2
 	github.com/ipfs/go-block-format v0.0.2
-	github.com/ipfs/go-cid v0.0.5
-	github.com/ipfs/go-datastore v0.4.4
+	github.com/ipfs/go-cid v0.0.7
+	github.com/ipfs/go-datastore v0.4.5
 	github.com/ipfs/go-detect-race v0.0.1
 	github.com/ipfs/go-ipfs-blockstore v0.1.4
 	github.com/ipfs/go-ipfs-blocksutil v0.0.1
 	github.com/ipfs/go-ipfs-delay v0.0.1
 	github.com/ipfs/go-ipfs-exchange-interface v0.0.1
 	github.com/ipfs/go-ipfs-routing v0.1.0
-	github.com/ipfs/go-ipfs-util v0.0.1
+	github.com/ipfs/go-ipfs-util v0.0.2
 	github.com/ipfs/go-log v1.0.4
 	github.com/ipfs/go-metrics-interface v0.0.1
 	github.com/ipfs/go-peertaskqueue v0.2.0
 	github.com/jbenet/goprocess v0.1.4
 	github.com/libp2p/go-buffer-pool v0.0.2
-	github.com/libp2p/go-libp2p v0.8.3
-	github.com/libp2p/go-libp2p-core v0.5.2
+	github.com/libp2p/go-libp2p v0.11.1-0.20201102192925-b33cbc714465
+	github.com/libp2p/go-libp2p-core v0.7.0
 	github.com/libp2p/go-libp2p-loggables v0.1.0
 	github.com/libp2p/go-libp2p-netutil v0.1.0
-	github.com/libp2p/go-libp2p-testing v0.1.1
-	github.com/libp2p/go-msgio v0.0.4
-	github.com/multiformats/go-multiaddr v0.2.1
-	github.com/multiformats/go-multistream v0.1.1
-	go.uber.org/zap v1.14.1
+	github.com/libp2p/go-libp2p-testing v0.3.0
+	github.com/libp2p/go-msgio v0.0.6
+	github.com/multiformats/go-multiaddr v0.3.1
+	github.com/multiformats/go-multistream v0.1.3-0.20201027223914-4e0841981936
+	go.uber.org/zap v1.16.0
 )
 
 go 1.12
```

Comment:
Does this need to get bumped too?

Reply:
I think this just got bumped due to the libp2p bump.

Reply:
Ah, you mean the go version? Eh, not really.